test/python/opflow/test_pauli_sum_op.py

```diff
@@ -164,7 +164,7 @@ def test_eval(self):
         """ eval test """
         target0 = (2 * (X ^ Y ^ Z) + 3 * (X ^ X ^ Z)).eval("000")
         target1 = (2 * (X ^ Y ^ Z) + 3 * (X ^ X ^ Z)).eval(Zero ^ 3)
-        expected = DictStateFn({"011": (2 + 3j)})
+        expected = DictStateFn({"011": (3 + 2j)})
```

Comment:
Why do you swap the real and imag values?

Reply:
I fixed the bug that the order is not aligned.
https://github.com/Qiskit/qiskit-terra/pull/5623/files#diff-7451487da776f4e8e4738bc39d578bc69c52b7314e40364df2532428c2e95ce6R266-R268